release notes/v0.44.0.md

```diff
@@ -0,0 +1,221 @@
+webvital_first_input_delay................: avg=300µs    min=200µs    med=300µs    max=399.99µs p(90)=379.99µs p(95)=389.99µs
+webvital_first_input_delay_good...........: 2       0.646663/s
+webvital_interaction_to_next_paint........: avg=16ms     min=16ms     med=16ms     max=16ms     p(90)=16ms     p(95)=16ms
+webvital_interaction_to_next_paint_good...: 1       0.323332/s
+webvital_largest_content_paint............: avg=303.6ms  min=141.1ms  med=303.6ms  max=466.1ms  p(90)=433.6ms  p(95)=449.85ms
+webvital_largest_content_paint_good.......: 2       0.646663/s
+webvital_time_to_first_byte...............: avg=205.23ms min=104.79ms med=188.39ms max=322.5ms  p(90)=295.67ms p(95)=309.08ms
+webvital_time_to_first_byte_good..........: 3       0.969995/s
+```
+
+You may have noticed other metrics in there too. We rely on the [web-vitals](https://github.com/GoogleChrome/web-vitals) JS library which exposes a few more metrics, so we've left them in for you to experiment with. You can find more details on all the browser module metrics in our [documentation](https://k6.io/docs/using-k6-browser/browser-metrics/).
+
+You will no longer see `browser_first_contentful_paint` in the summary, and instead you can work with `webvital_first_contentful_paint`.
+
+### UX improvements and enhancements
+
+- [#2906](https://github.com/grafana/k6/pull/2906) Added multiple date-time formats for CSV output. Thanks, @Azanul!
+- [#2916](https://github.com/grafana/k6/pull/2916) k6 started to show the actual binary's name in the usage help. Thanks, @ariasmn!
+- [#2942](https://github.com/grafana/k6/pull/2942) Reference 'k6 cloud' instead of 'Load Impact' in docs and errors.
+- [#2985](https://github.com/grafana/k6/pull/2985) Added support of async functions for `setup` and `handleSummary`.
+- [#2901](https://github.com/grafana/k6/pull/2901) Added a warning when the number of time series exceeds 100 000, which could lead to potential out-of-memory issues.
+- [#2997](https://github.com/grafana/k6/pull/2997) Added a new exit code (`109`), used on a go panic.
+- [browser#788](https://github.com/grafana/xk6-browser/pull/788) Updated the browser readme to highlight that it is now a module in k6.
+- [browser#803](https://github.com/grafana/xk6-browser/pull/803) Users are now warned if the `browser.close` method is called more than once.
+- [browser#820](https://github.com/grafana/xk6-browser/pull/820) Added error handling to wildcard selectors, which cleans up the error output in the terminal.
+- [browser#843](https://github.com/grafana/xk6-browser/pull/843) Removed the build step from github actions. From this release onwards, no new standalone browser binaries will be built and available from the [releases](https://github.com/grafana/xk6-browser/releases) section. The latest version of the browser module will be available in the k6 binary which can be found in the [k6 releases](https://github.com/grafana/k6/releases) page.
```

Comment:
Maybe we should mention this in the Breaking changes section? Whoever was relying on the xk6-browser prebuilt binaries will need to be aware that they should use the k6 binary instead.

Reply:
Yeah, that 's a better place for it: 96db17a4d908d0da7577afe7ed270ce340078931